metrics/prometheus/linux/prometheus.yml

```
@@ -8,6 +8,6 @@ scrape_configs:
     static_configs:
       - targets: ['localhost:9090']
 
-  - job_name: 'service-collector'
+  - job_name: 'mining-node-collector'
```

```suggestion
  - job_name: 'snarkOS-metrics-collector'
```
metrics can be collected from non-miners as well; this applies to `mac` and `windows` config files

---

updated to `snarkos-metrics-collector` for all configs.